cmd/influxd/run/server_suite_test.go

```diff
@@ -19,13 +19,13 @@ func init() {
 			&Query{
 				name:    "create database should succeed",
 				command: `CREATE DATABASE db0`,
-				exp:     `{"results":[{}]}`,
+				exp:     `{"results":[{"statement_id":0}]}`,
```

Comment:
No, the `statement_id` is pretty important. It tells the client which statement that empty response is for so it knows it ran. Otherwise, there's no way to know which statement that empty response references.